main/guides/zoe/contracts/otc-desk.md

```diff
@@ -7,9 +7,9 @@
 
 ![Building a Composable DeFi Contract](./assets/title.jpg)
 
-This is the OTC Desk contract from the ["Building a
+This is the OTC Desk contract from the "Building a
 Composable DeFi Contract" episode of Cosmos Code With
-Us](https://cosmos.network/series/code-with-us/building-a-composable-defi-contract).
+Us workshop.
```

Comment:
rather than removing this link, I suggest pointing it to https://www.youtube.com/watch?v=e9dMkC2oFh8

Reply:
Done.